design-proposals/enhancements.md

```diff
@@ -0,0 +1,109 @@
+# Overview
+
+Rework the existing KubeVirt design proposal process into a more
+formal - but still lean - and SIG-aligned Enhancement Proposal process.
+
+## Motivation
+
+Today it's a [very small group](https://github.com/kubevirt/community/blob/main/OWNERS#L7-L14)
+which can approve design-proposals.
+While certain individuals can approve design proposals, once they are merged
+it is not clear who is owning them post-merge.
+
+In this proposal we are proposing to change the process and shape them
+around SIGs.
+
+At its core, this proposal requires:
+
+1. designs to be sponsored by a SIG
+2. to make one SIG responsible for the design process (reviews of design, code, documentation)
+3. to make one SIG responsible for managing the design process (collaboration with other SIGs as needed)
+4. to make one SIG to own the feature once it has been merged. The SIG is responsible for maintaining, fixing, running, _everything-it_.
```

Comment:
Shouldn't this process encourage a VEP author to become member of said SIG? Currently it sounds a bit like as a VEP author I can forget about the feature once it is merged and the SIG has to carry the burden of maintaining it.

Reply:
> Currently it sounds a bit like as a VEP author I can forget about the feature once it is merged and the SIG has to carry the burden of maintaining it.

Do you mean "once it GAs"?
If you'd forget about the feature before it GAs, it will eventually be dropped.

Forgetting about a feature after developing it to the point it GAs sounds fair to me TBH.

> I would add that the original author is encouraged but not required to become a member of the SIG to help with on-going maintenance of the feature they are introducing.

I'm not against encouraging it, but not sure it's necessary TBH. These scenarios are completely valid IMO:
1. Someone develops a feature under a certain SIG but is interested only in the feature's narrow scope, not the entire SIG. For example, someone can develop a network feature under sig-network, but care about this specific feature and not about SIG network generally.
2. Someone can develop a feature, own and maintain it for a while, then stop maintaining it (due to switching his position, becoming interested in other stuff, etc). IOW, we should expect that people will move around. In any case the SIG always owns all of its features while people can stop maintaining them.